src/Geopilot.Frontend/cypress/e2e/app.cy.js

```diff
@@ -75,4 +75,62 @@ describe("General app tests", () => {
     selectLanguage("it");
     cy.contains("Clicca per selezionare");
   });
+
+  it("displays correct localized application name when language changes", () => {
+    // Intercept the client-settings.json request to dynamically extract the values
+    cy.intercept("**/client-settings.json").as("clientSettings");
+
+    // Visit the home page
+    cy.visit("/");
+
+    // Wait for client settings to load and extract the localNames
+    cy.wait("@clientSettings").then(interception => {
+      // Extract the application settings from the intercepted response
+      const settings = interception.response.body;
+      const localNames = settings.application.localName;
+
+      // Wait for the language selector to load
+      cy.wait(500);
```

Comment:
Q: Braucht es dieses `wait`? Das ist eigentlich ein Anti-Pattern und sollte nur in Ausnahmefällen verwendet werden, siehe https://docs.cypress.io/app/core-concepts/best-practices#Unnecessary-Waiting.